package.json

```diff
@@ -17,8 +19,9 @@
     "pretty-quick": "^2.0.1"
   },
   "scripts": {
+    "build": "tsc",
     "develop": "nodemon src/app.js",
-    "start": "node src/app.js",
+    "start": "node --experimental-modules --experimental-json-modules src/build/app.js",
```

Comment:
I feel like there should be a better way to do this than using these experimental flags.  My (probably somewhat incorrect) understanding is either: 
1. Typescript needs to compile the imports into requires
2. We need to have node read the import syntax (aka the experimental flags)